README.md

```diff
@@ -1,88 +1,129 @@
+<img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="400">
+
 ## Amplify for Android (Preview)
-<img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="550">
 
-AWS Amplify provides a high-level interface to perform different categories of cloud operations. Each category can be implemented by a _plugin_. Our default plugins support interaction with Amazon Web Services (AWS), but AWS Amplify is designed to be extensible to any other backend or service.
+AWS Amplify provides a high-level interface to perform different
+categories of cloud operations. Each category is fulfilled by a
+_plugin_. You specify which plugins to use during setup.
+
+The default plugins that we provide are designed to facilitate
+interaction with Amazon Web Services (AWS). But, the Amplify framework
+is designed to be extensible to any other backend or service -- even a
+competing cloud provider!
+
+If you're new to the project, checkout the
+[Getting Started Guide](https://docs.amplify.aws/lib/getting-started/setup/q/platform/android).
+
+## The Categories
+
+### [DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/android):
+
+Model your app's data. Save, query, and observe changes to your data
+from a local repository. Let DataStore synchronize your local data with
+the Cloud. Our default implemenation syncs local data to/from a DynamoDB
+database, via an AppSync front-end.
+
+### [REST APIs](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android):
+
+Easy auth and request signing against multiple REST endpoints. Our
+default plugin works great with Amazon API Gateway.
+
+### [GraphQL APIs](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/android)
+
+Data modeling and simple auth against GraphQL endpoints. Our default
+plugin targets Amazon AppSync.
```

Comment:
Nit: AWS AppSync, but since you mentioned it above, just "AppSync" will suffice.